src/components/FavoriteServices/ServiceIcon.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PlaceholderIcon from '../AllServicesDropdown/icon-placeholder';
+import ACSIcon from '../AllServicesDropdown/icon-acs';
+import AnsibleIcon from '../AllServicesDropdown/icon-ansible';
+import AppServicesIcon from '../AllServicesDropdown/icon-app-services';
+import DataScienceIcon from '../AllServicesDropdown/icon-data-science';
+import EdgeIcon from '../AllServicesDropdown/icon-edge';
+import InsightsIcon from '../AllServicesDropdown/icon-insights';
+import OpenShiftIcon from '../AllServicesDropdown/icon-openshift';
+import QuayIoIcon from '../AllServicesDropdown/icon-quay-io';
+import RHIcon from '../AllServicesDropdown/icon-rh';
+import ServicesIcon from '../AllServicesDropdown/icon-services';
+import SubscriptionsIcon from '../AllServicesDropdown/icon-subscriptions';
+import TrustedContentIcon from '../AllServicesDropdown/icon-trusted-content';
+
+const iconEnum: { [key: string]: React.ComponentType } = {
+  ACSIcon: ACSIcon,
```

Comment:
Create an enum of available icons and use that as a key instead of a string.

Reply:
Fixed